src/main/start-main-application/runnables/setup-sentry.injectable.ts

```diff
@@ -3,21 +3,19 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import { getInjectable } from "@ogre-tools/injectable";
-import { initializeSentryReporting } from "../../../common/sentry";
 import { init } from "@sentry/electron/main";
 import { beforeElectronIsReadyInjectionToken } from "../runnable-tokens/before-electron-is-ready-injection-token";
+import initializeSentryReportingWithInjectable from "../../../common/error-reporting/initialize-sentry-reporting.injectable";
 
 const setupSentryInjectable = getInjectable({
```

Comment:
The injectable should still be `causesSideEffects: true`, because it accesses `init` from electron.

Reply:
Ah I see okay